src/modules/taskHandler.js

```diff
@@ -0,0 +1,68 @@
+import updateStatus from './taskUpdater.js';
+import taskDrag from './taskDrag.js';
+
+const listContainer = document.getElementById('to-do-list');
+
+const display = (tasksObject) => {
+  const tasks = tasksObject.getTasks();
+  taskDrag();
+  listContainer.innerHTML = '';
+  for (let i = 0; i < tasks.length; i += 1) {
+    const listItem = document.createElement('li');
+    listItem.classList.add('list-item');
+    listItem.classList.add('draggable');
+    listItem.setAttribute('draggable', 'true');
+    listItem.innerHTML = `<div class="checkbox" id="checkbox${i}"></div>
+                          <img class="checkmark" id="checkmark${i}" src="./asset/checkmark.svg" alt="Checkmark">
+                          <div class="input-div" id="input-div${i}">
+                            <input type="text" class="task-description" id="description${i}">
+                            <img class="remove" id="remove${i}" src="./asset/garbage.svg" alt="Garbage bin icon">
+                            <img class="dots" draggable="false" id="dots${i}" src="./asset/vertical-dots.svg" alt="Vertical dots">
+                          </div>
+                          `;
+    listContainer.appendChild(listItem);
+    const checkbox = document.getElementById(`checkbox${i}`);
+    const checkmark = document.getElementById(`checkmark${i}`);
+    const text = document.getElementById(`description${i}`);
+    const elementArray = [checkmark, checkbox, text];
+    updateStatus(elementArray, tasksObject, i);
+    const remove = document.getElementById(`remove${i}`);
+    const dots = document.getElementById(`dots${i}`);
+    const input = document.getElementById(`description${i}`);
+    const inputDiv = document.getElementById(`input-div${i}`);
+    input.value = tasks[i].description;
+
+    document.addEventListener('click', (event) => {
+      const isClickInside = inputDiv.contains(event.target);
+      if (!isClickInside) {
+        // the click was outside the specifiedElement, do something
+        listItem.style.backgroundColor = '#fff';
+        input.style.backgroundColor = '#fff';
+        remove.style.display = 'none';
+        dots.style.display = 'block';
+        if (checkmark.style.display === 'block') text.style.textDecoration = 'line-through';
+      } else {
+        listItem.style.backgroundColor = '#fffeca';
+        input.style.backgroundColor = '#fffeca';
+        remove.style.display = 'block';
+        dots.style.display = 'none';
+        if (checkmark.style.display === 'block') text.style.textDecoration = 'none';
+      }
```

Comment:
That's indeed better yea I will do it.

Reply:
Thank you for pointing that out @mirouhml. 
There is a great debate over this topic. JS is the most expensive part of your website. I believe using one 2 classes styling for all elements and another 2 for display none and display block or flex would be more efficient in the long run. Imagine you will need to change 10 or 100 items and also have more styling. This would be very costly for your application if handled solely with JS.

But, as I mentioned above, it's a great debate, that's why I left it as a complimentary suggestion especially that it doesn't have that much effect on a small application such as this

Please, let me know if you have any further questions, and in the meanwhile, please check this article for more insights about [the cost of JS](https://medium.com/@addyosmani/the-cost-of-javascript-in-2018-7d8950fbb5d4) compared to HTML and CSS. ^_^